boards/riscv/litex_vexriscv/litex_vexriscv_defconfig

```diff
@@ -11,19 +11,7 @@ CONFIG_LITEX_TIMER=y
 CONFIG_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_CONSOLE=y
-CONFIG_NETWORKING=y
-CONFIG_ETH_LITEETH=y
-CONFIG_ETH_LITEETH_0=y
```

Comment:
not sure you can remove CONFIG_ETH_LITEETH_0/CONFIG_ETH_LITEETH (may need to move to Kconfig.defconfig

Reply:
As it is done often, it's the application/sample that is responsible for that when SoC doesn't select such defaults in SoC Kconfig.defconfig files. This should be properly fixed by using the `DT_HAS*` helpers.

Reply:
Agreed this should move to DT_HAS_*, but for now is this not breaking anything?

Reply:
Moved options to `Kconfig.defconfig`, please check latest changeset. Done at board level (not SoC level) as not all boards may want such behavior if NETWORKING=y. I've seen quite a few boards doing similar thing for the L2 option.